README.md

```diff
@@ -1,12 +1,11 @@
 # Job Manager
 
 [![CircleCI branch](https://img.shields.io/circleci/project/github/DataBiosphere/job-manager/master.svg?label=Tests%20on%20Circle%20CI&style=flat-square)](https://circleci.com/gh/DataBiosphere/job-manager/tree/master)
-![Github](https://img.shields.io/badge/Supported%20Backends-cromwell%20%7C%20dsub-blue.svg?&style=flat-square)
+![Github](https://img.shields.io/badge/Supported%20Backends-cromwell-blue.svg?&style=flat-square)
```

Comment:
I'm surprised that cromwell-sub is still showing.

![image](https://user-images.githubusercontent.com/484484/128036032-0dab67c3-828a-4007-b06a-3599f511a25a.png)

Reply:
I think that only updates after merge

Reply:
You can preview it by checking out the branch and viewing in IntelliJ markdown split view

Reply:
Got it, thanks!